src/rust/ide/view/graph-editor/src/component/breadcrumbs/project_name.rs

```diff
@@ -30,6 +31,8 @@ use logger::AnyLogger;
 
 /// Project name used as a placeholder in `ProjectName` view when it's initialized.
 pub const UNKNOWN_PROJECT_NAME:&str = "Unknown";
+/// Default line height for project names.
+pub const LINE_HEIGHT : f32 = TEXT_SIZE * 1.5;
```

Comment:
here you are ussing spaces around `:`, the line above not. Use one style :) (In public consts I believe the spaced one is much better, we use it everywhere)